libs/utils/utils.js

```diff
@@ -260,6 +260,14 @@ export function isInTextNode(node) {
   return node.parentElement.firstChild.nodeType === Node.TEXT_NODE;
 }
 
+export function elContainsText(el) {
```

Comment:
If this method is only used in decorate and only a specific method within decorate, should it stay in there?

Reply:
I guess I thought it was generic enough that could be used elsewhere in the future, I guess if that happens we can move it to utils then. For now I will move back to decorate. Thanks